From 5dad44a8c1d2fedb18e7744f48e1512246d8ecc5 Mon Sep 17 00:00:00 2001
From: "Srivatsa S. Bhat (VMware)" <srivatsa@csail.mit.edu>
Date: Fri, 31 Mar 2023 14:31:12 -0700
Subject: [PATCH] linux-rt, Makefile: Add kernel flavor info to the generated
 version.h file

Add "PHOTON_KERNEL_FLAVOR_RT" macro definition to the generated file
include/generated/uapi/linux/version.h to identify Photon's linux-rt
kernel flavor.

Signed-off-by: Srivatsa S. Bhat (VMware) <srivatsa@csail.mit.edu>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index db5b181a3..d88dae999 100644
--- a/Makefile
+++ b/Makefile
@@ -1270,7 +1270,7 @@ define filechk_version.h
 	echo '#define PHOTON_KERNEL';                                                   \
 	echo '#define PHOTON_KERNEL_RELEASE                                             \
 	$(shell echo -n "$(KERNELRELEASE)" | cut -d'-' -f2 | cut -d'.' -f1)';           \
-	echo '#define PHOTON_KERNEL_FLAVOR_'
+	echo '#define PHOTON_KERNEL_FLAVOR_RT'
 endef
 
 $(version_h): private PATCHLEVEL := $(or $(PATCHLEVEL), 0)
-- 
2.39.4

